src/test/txpackage_tests.cpp

```diff
@@ -327,4 +327,122 @@ BOOST_FIXTURE_TEST_CASE(package_submission_tests, TestChain100Setup)
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(tx_child->GetHash())));
     }
 }
+
+// Tests for packages containing transactions that have same-txid-different-witness equivalents in
+// the mempool.
+BOOST_FIXTURE_TEST_CASE(package_witness_swap_tests, TestChain100Setup)
+{
+    LOCK(cs_main);
+
+    // Transactions with a same-txid-different-witness transaction in the mempool should be swapped.
```

Comment:
I think the word swapped is slightly confusing here. It suggests to me that child1 will be replaced ("swapped") with child2 in the mempool.

Reply:
Changed result enum to `DIFFERENT_WITNESS` and verb from "swapped" to "ignored" in comments